pkgs/applications/radio/qdmr/default.nix

```
@@ -0,0 +1,67 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeDesktopItem,
+  writeText,
+  cmake,
+  wrapQtAppsHook,
+  libusb1,
+  libyamlcpp,
+  qtlocation,
+  qtserialport,
+  qttools,
+  qtbase,
+}:
+stdenv.mkDerivation rec {
+  pname = "qdmr";
+  version = "0.11.0";
+
+  src = fetchFromGitHub {
+    owner = "hmatuschek";
+    repo = "qdmr";
+    rev = "v${version}";
+    sha256 = "sha256-N8s2Qva9PNu8fXcMNpLnhYpVgol58Ieh4vC95YJw0Yc=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    libyamlcpp
+    libusb1
+    qtlocation
+    qtserialport
+    qttools
+    qtbase
+  ];
+
+  meta = {
+    lib.description = "A codeplug programming tool for DMR radios";
+    lib.homepage = "https://dm3mat.darc.de/qdmr/";
+    lib.license = lib.licenses.gpl3;
+    lib.maintainers = [ lib.maintainers.janik ];
+    lib.mainProgram = "qdmr";
```

```suggestion
    description = "A codeplug programming tool for DMR radios";
    homepage = "https://dm3mat.darc.de/qdmr/";
    license = lib.licenses.gpl3;
    maintainers = [ lib.maintainers.janik ];
    mainProgram = "qdmr";
```

interesting that ofborg doesn't complain about this…

We're also trying to phase out licenses like `gpl3`, because they're imprecise. This programs seems to be `gpl3Plus` meaning, GPL version 3 or later.

---

I just pushed the suggested changes myself so the commit history stays clean

---

jup would also say it's GPL version 3 so if you want i can change the license to gpl3Plus in the meta information

---

https://github.com/hmatuschek/qdmr/blob/master/LICENSE the license file in the git repo says it's gpl3

---

Exactly. https://github.com/hmatuschek/qdmr/blob/master/LICENSE#L572 and https://github.com/hmatuschek/qdmr/blob/9c9eb5d1e7a8b382eba9a1edaedc36a403221240/README.md#license "either version 3 of the License, or (at your option) any later version" hence `Plus`.

---

Made it ``` gpl3Plus ``` is there anything else?

---

You can also ask for clarification upstream, they might be more knowledgeable about this.

---

Okay I just asked in the [qdmr-matirx](https://matrix.to/#/#qdmr:darc.de) the developer confirmed that the license is GPLv3+

---

Yes I did.